apps/omg_watcher_info/test/omg_watcher_info/db/block_test.exs

```diff
@@ -32,6 +32,31 @@ defmodule OMG.WatcherInfo.DB.BlockTest do
     end
   end
 
+  describe "get/1" do
+    @tag fixtures: [:initial_blocks]
+    test "retrieves a block by block hash" do
+      hash = "##{1000}"
+      block = hash |> DB.Block.get()
+
+      assert block.__struct__ == DB.Block
```

Comment:
Is assert block = %DB.Block{} more idiomatic Elixir?

Example from O's PR: `assert %OMG.Utils.Paginator{} = results`
https://github.com/omisego/elixir-omg/pull/1192/files#diff-749f6bdeafde2c31418e3c125d44f010R61

Reply:
Yup! `.__struct__` is Elixir's internal implementation (which may change without notice) so it is discouraged.

Reply:
Thanks for that! Changed in 04cd40f.